doc/mgr/dashboard.rst

```diff
@@ -282,7 +276,7 @@ wanted or required. See :ref:`dashboard-proxy-configuration` for more details.
 
 .. note::
 
-  You need to restart the Ceph manager processes manually after changing the SSL
+  You need to restart Ceph manager processes manually after changing the SSL
```

Comment:
"You must manually restart..."

Reply:
`To boldly go`.  I think `manually` actually doesn't add to the sentence, pushing my idea, PTAL